requirements.txt

```
@@ -7,3 +7,4 @@ h5py==2.10.0
 gym==0.17.2
 kornia==0.4.0
 Cython==0.29.21
+stable-baselines3==0.10.0
```

```suggestion
stable-baselines3>=0.10.0
```
(as we are currently in beta, asking for the latest version is recommended)